### ISSUE TITLE
Scrolling logs scrolls them outside of visible area

### ISSUE THREAD

**Issue by gigor** [warpdotdev/Warp#4830]
### Describe the bug

I have a Vite dev server running; on save, the logs are scrolled outside of the visible area.

### To reproduce

1. Start a webserver or any continuous task, that will periodically write.
2. Set scroll to bottom of block
3. Push some logs
4. The window will be automatically scrolled and you will only see the last line of your log

### Expected behavior

When scroll to bottom of block is enabled, scroll in a way that alligns the bottom of the block with the bottom of visible area, not the top.

### Screenshots

![CleanShot 2024-04-26 at 13 11 55@2x](https://github.com/warpdotdev/Warp/assets/2462955/9c6fe908-05b4-455e-8d09-7cba3700705f)

https://github.com/warpdotdev/Warp/assets/2462955/31659c12-ef8b-49d9-9649-8a8e966d92e1

### Operating system

MacOS

### Operating system and version

13.6.6

### Current Warp version

v0.2024.04.23.08.01.stable_03

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by dannyneira** [comment]
We're tracking this issue over at #610  and would appreciate it if you could 👍 and comment there if you're also experiencing the same bug. This helps us gauge the impact of the issue. 

We'll be closing this as a duplicate.

**Comment by dannyneira** [comment]
Duplicate of #610

**Comment by gigor** [comment]
@dannyneira, the issue you are referring to is two years old and locked. I read it as there is no chance it will be fixed, and it's a deal breaker for me.

**Comment by dannyneira** [comment]
We did some early work on the request, and there is an option for more scrollback in our dev build, but the performance/memory consumption issues it introduced need to be worked out before we can release it to stable.
